src/bundle/Resources/config/services/ui_config/common.yaml

```
@@ -104,6 +104,10 @@ services:
 
     Ibexa\Bundle\AdminUi\Templating\Twig\EmbeddedItemEditFormExtension: ~
 
+    Ibexa\Bundle\AdminUi\Templating\Twig\SecurityExtension:
+        arguments:
+                $csrfTokenIntention: 'rest'
```

We don't need to hardcode the token: 

```suggestion
                $csrfTokenIntention: '%ibexa.rest.csrf_token_intention%'
```

The only thing is to ensure `ibexa/rest` is dependency for `ibexa/admin-ui` (should be)